shell/pom.xml

```diff
@@ -125,9 +125,9 @@
       </plugin>
       <plugin>
         <artifactId>maven-resources-plugin</artifactId>
-      </plugin>
-      <plugin>
-        <artifactId>maven-shade-plugin</artifactId>
+        <configuration>
```

Comment:
what's the effect of this change?

Reply:
It seems the whole `sh` dir disappears in the `interpreter` folder, not only "configurations", is it expected?

Reply:
Yes, correct. Shell interpreter was and will be removed from the official binary because it lead a lot of security issues.

Reply:
LGTM if it's the expected behavior. Test may be need to adjusted to adapt this change.